data/lib/src/datasource_impl/document_datasource_impl.dart

```diff
@@ -47,7 +47,7 @@ class DocumentDataSourceImpl implements DocumentDataSource {
 
   @override
   Future<FileUploadState> upload(FileInfo fileInfo, Token token, Uri baseUrl) async {
-    File file = File(fileInfo.filePath + fileInfo.fileName);
+    var file = File(fileInfo.filePath + fileInfo.fileName);
```

Comment:
I think it can be `final` instead of `var`. same to others in this commit.

Reply:
ok